test/e2e/commands.js

```diff
@@ -112,7 +112,20 @@ function testContainsOrNotExists(target, value) {
   }
 }
 
+function testQuestionButtons(buttonsObject = {btn__action: 'Submit', btn__feedback: 'Show feedback'}) {
+  Object.keys(buttonsObject).forEach((key) => {
+    cy.get(`.${key}`).should('contain', buttonsObject[key])
+  })
+}
+
+function stripHtml(text) {
+  const textWithoutHtml = text.replace(/<[^>]*>/g, '');
+  cy.wrap(textWithoutHtml).as('text');
+}
```

Comment:
Could this work as a simple return without the need to involve Cypress complexity?

```suggestion
function stripHtml(text) {
  return text.replace(/<[^>]*>/g, '');
}
```

It may be that we need to move it into a helpers file outside of the test files and reference it from there?
It just feels like the Cypress wrapper is more than would be necessary

Reply:
Upon discussion, moving to the helpers file would mean touching core and in a quite confusing way.
Disregarding the above suggestion

Reply:
Did you agree that the wrapper context is necessary? 

Using `const stripped = this.text` rather than `const stripped = cy.stripText()` seems really janky. The `this` variable is for context and not for parsing local variables.

Reply:
Yes it is necessary. Within cypress setting a const to a function doesn't work, even if that function returns a variable. I would prefer to do that of course, but it is not possible unfortunately.

Reply:
If you define it as:

`test/e2e/helpers.js`
```js
cy.helpers = {
  stripHtml(text) {
    let tmp = document.createElement("DIV");
    tmp.innerHTML = text;
    const textWithoutHtml = tmp.textContent || tmp.innerText || "";
    return textWithoutHtml
  }
}
```

Then at the top of the `commands.js` file put:
```js
import './helpers` 
```

You should be able to refer to it as:
```js
      const bodyWithoutHtml = cy.helpers.stripHtml(textInputComponent.body);
      cy.testContainsOrNotExists('.textinput__body', bodyWithoutHtml)
```

Reply:
I personally think this is more janky to have a separate file for this but I'll add it

Reply:
It doesn't need to be a separate file if you prefer. I don't mind. I guess if we add more we can reorganise at that point.